docs/resources/credential_issuance_rule.md

```diff
@@ -75,12 +75,12 @@ resource "pingone_credential_issuance_rule" "my_credential_issuance_rule" {
 
 - `automation` (Attributes) Contains a list of actions, as key names, and the update method for each action. (see [below for nested schema](#nestedatt--automation))
 - `credential_type_id` (String) Identifier (UUID) of the credential type with which this credential issuance rule is associated.
-- `digital_wallet_application_id` (String) Identifier (UUID) of the customer's Digital Wallet App that will interact with the user's Digital Wallet.
```

Comment:
Just to check this is meant to be now an optional attribute?

Reply:
Yes, it should be optional (ref: https://apidocs.pingidentity.com/pingone/platform/v1/api/#credential-issuance-rules).  I realized it was inconsistent with the schema.

However, I need to add a validation rule.  If the `status` is `ACTIVE`, the `digital_wallet_application_id` should be present otherwise the API will return an error.  If `status` is `DISABLED`, the value is optional.  Corrections and testing in progress.

Reply:
After testing, identified the API documentation is incorrect. Even with a `status` of `DISABLED`, the `digital_wallet_application_id` is required.  

```
│ Error: Error when calling `CreateCredentialIssuanceRule`: Validation Error : [digitalWalletApplication must have an id]
│
│   with pingone_credential_issuance_rule.credential1_issuance_rule,
│   on wtf.tf line 410, in resource "pingone_credential_issuance_rule" "credential1_issuance_rule":
│  410: resource "pingone_credential_issuance_rule" "credential1_issuance_rule" {
│
│ PingOne Error Details:
│ ID: 2c587291-0f2f-44b4-9112-97076a565414
│ Code: INVALID_DATA
│ Message: Validation Error : [digitalWalletApplication must have an id]
│ Details object: [{"code":"INVALID_VALUE","message":"must have an id","target":"digitalWalletApplication"}]
```

Reply:
Resolved. Schema for `dsigital_wallet_application_id` is reverted to required per review and testing.